Sora

black-calculus opened the pull request:
Resolves #540 .
Big batch of Sora changes for the next beta. Got one last final coming up and a lot more characters to go through and put up PRs for so I kinda rushed through this changelog, hopefully it's not missing anything.
[Sora Changelog.txt](https://github.com/HDR-Development/HewDraw-Remix/files/8608384/Sora.Changelog.txt)

You can grab the animations and motion_list you need from the [latest nightly romfs release](https://github.com/HDR-Development/romfs-release/releases/tag/v0.16.0).

black-calculus committed:
Fix momentum conservation on nair/fair, fix dair DJCs, readjust nair and fair momentum handling, make fair 2 cancels less busted

Wish325 committed:
Sora Nair, Dair, Param changes

As the title suggest.

Wish325 committed:
Even More Sora Changes

im a nerd

Wish325 committed:
forgot a bair thing

forgot this bair thing, frame 10

Wish325 committed:
Various Sora Fixes

Down Smash: Knockback Growth Increased for nerfing grounded combos.

F-Smash Bug Fix

Down-Air, no longer reverse hits, absolutely required change for the move to function.

black-calculus committed:
Merge branch 'dev' into sora

black-calculus committed:
Merge pull request #535 from Wish325/sora

Merging in WIP changes from Wish's fork

black-calculus committed:
Sora rework part 2

black-calculus committed:
Utilt readjustment

black-calculus committed:
Clean up dsmash script

Wish325 committed:
Merge branch 'dev' into sora

jobrien97 commented on fighters/trail/src/opff.rs:
```diff
@@ -34,23 +86,54 @@ unsafe fn fair_cancels(boma: &mut BattleObjectModuleAccessor, cat1: i32, status_
+    if boma.is_status(*FIGHTER_STATUS_KIND_SPECIAL_HI){
+        if MotionModule::frame(boma) > 37.0 {
+            if AttackModule::is_infliction(boma, *COLLISION_KIND_MASK_HIT) {
+                DamageModule::add_damage(boma, 1.0, 0);
```
Why are we adding damage here?

jobrien97 reviewed (changes requested):
remove printlns pls

MadVillainLordQ commented:
Overall:
Really well done from a design standpoint. The following are notes/suggestions to his moveset

Notes:
- At some point, Firaga should be changed to not halt vertical momentum. It is extremely strong to be able to shoot a projectile at ground level and waveland immediately afterwards
- Add reverse hit properties on the new dair, specifically on frames 8-9 as this style of hitbox is a catch all move and removes the potential nuance of DI when hitting an opponent. 
- The suggestion would be to make frames 8-9 reverse, 10-12 hit normally, and frame 13 be the sole spike hitbox and only place the spike on the blade of kingdom key, not on the hilt or body of sora. Another idea is to have only frame 8 be the reverse, 9-12 regular, and 13 be spike (with the aforementioned hitbox placement)
- It has been mentioned in discord channels but buffering short hop neutral b with thunder causes grounded thunder to appear instead of aerial thunder
- I have concerns on the potential looping of fair 1/2 as they are +22 and +15 on hit at 0. Hitstun should be reduced on those moves to avoid that.
- Firaga hitstun is a tad high at 34 frames. Being able to waveland out of it makes it have much more frame advantage than its current form. It should be brought down to be +24 to be more inline with moves that have similar properties like wolf laser (+22)
- Uair damage is a concern of mine as the move is already very good at comboing and is further improved with the attribute changes that sora has had. I believe it should be brought down to 10 damage to be more inline with other arc sword swipes of similar nature
- Awesome change with the proration of moves after fair 2 cancels. I do think the moves have to have even more damage reduction though. 
- It would be nice if there was some sort of visual/audio cue when you hit the spike on dair. Maybe like the little ding noise from his stop taunt could work

black-calculus committed:
Fix self damage on fair 2 -> nair cancels

Wish325 committed:
Final Sora Adjustments

This is the last update ill be pushing to Sora's numbers for this patch, Adjusted aerials, spells and some params. Have updated changelog to reflect this.

Wish325 committed:
Merge branch 'sora' of https://github.com/HDR-development/HewDraw-Remix into sora

black-calculus committed:
Merge branch 'dev' into sora

Wish325 committed:
Param Stuff

Param changes that i didnt do in commit the other day by accident

black-calculus committed:
Merge pull request #598 from Wish325/sora

Last Sora Adjustments

black-calculus committed:
Fix self damage on up b hit actionability
